charts/kong/README.md

```diff
@@ -581,7 +581,9 @@ namespaces. Limiting access requires several changes to configuration:
 Setting `deployment.daemonset: true` deploys Kong using a [DaemonSet
 controller](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/)
 instead of a Deployment controller. This runs a Kong Pod on every kubelet in
-the Kubernetes cluster.
+the Kubernetes cluster. For such configuration it may be desirable to configure
+Pods to use the network of the host they run on instead of a dedicated network
+namespace. It can be achieved by setting `deployment.hostNetwork: true`.
```

Comment:
Wouldn't `HostPort` Service be a more idiomatic way to go with that?

Reply:
I added more description - inspired by [this](https://kubernetes.github.io/ingress-nginx/deploy/baremetal/#via-the-host-network). This is why I wrote `it may be desirable`. PTAL